v1/tarball/write.go

```diff
@@ -49,6 +59,33 @@ func Write(p string, tag name.Tag, img v1.Image, wo *WriteOptions) error {
 	tf := tar.NewWriter(w)
 	defer tf.Close()
 
+	tr := tar.NewReader(Write(tag, img, wo))
```

Comment:
You should be able to just `io.Copy(f, r)` I think right?

Reply:
I thought so at first, but the copy doesn't seem to read correctly. Not sure if this is because it's a tar reader or if it's coming from a pipe, but if you have suggestions I'm all ears

Reply:
-1 for the pun 😆 

you're right about the extra read/write wrapping, but yeah this definitely wasn't the best construction. fixed up to make a little bit more sense